scipy/optimize/optimize.py

```diff
@@ -1535,6 +1541,16 @@ def _minimize_newtoncg(fun, x0, args=(), jac=None, hess=None, hessp=None,
             print("         Function evaluations: %d" % fcalls[0])
             print("         Gradient evaluations: %d" % gcalls[0])
             print("         Hessian evaluations: %d" % hcalls)
+    elif k2 >= maxiter:
+        warnflag = 3
+        msg = "Warning: CG iterations didn't converge."
```

Comment:
I would add a comment that the hessian may be incorrect. Mine actually was and this bug helped me track the error :)